dashboard/db/migrate/20160430111030_add_kind_to_survey_results.rb

```diff
@@ -0,0 +1,13 @@
+class AddKindToSurveyResults < ActiveRecord::Migration
+  def change
+    add_column :survey_results, :kind, :string, :after => :user_id
```

Comment:
By default, the `kind` column would become the last column (field) in the DB (model). The `:after => :user_id` makes it follow the `user_id` column (it seems natural for it to precede the `properties` column).